report/mapping.go

```diff
@@ -77,11 +82,28 @@ func ProcessName(_ string, m NodeMetadata) (MappedNode, bool) {
 	}, show
 }
 
-// ProcessContainer maps topology nodes to the containers they run in. We
+// ProcessEndpoint2Container maps endpoint topology nodes to the containers they run in. We
 // consider container and image IDs to be globally unique, and so don't scope
 // them further by e.g. host. If no container metadata is found, nodes are
 // grouped into the Uncontained node.
-func ProcessContainer(_ string, m NodeMetadata) (MappedNode, bool) {
+func ProcessEndpoint2Container(_ string, m NodeMetadata) (MappedNode, bool) {
+	var id, major, minor, rank string
+	if m["docker_container_id"] == "" {
+		id, major, minor, rank = "uncontained", "Uncontained", "", "uncontained"
+	} else {
+		id, major, minor, rank = m["docker_container_id"], "", m["domain"], ""
+	}
+
+	return MappedNode{
+		ID:    id,
+		Major: major,
+		Minor: minor,
+		Rank:  rank,
+	}, true
+}
+
+// ProcessContainer2Container maps container topology node to container mapped nodes.
+func ProcessContainer2Container(_ string, m NodeMetadata) (MappedNode, bool) {
```

Comment:
- "Process" is a vestige of the old topology naming scheme, let's drop it
- This is effectively an identity mapping, right? Can we call it ContainerIdentity?